.changeset/slow-starfishes-melt.md

```diff
@@ -0,0 +1,5 @@
+---
+"@paprika/build-translations": major
+---
+
+Use "fs" async API to avoid race condition
```

Comment:
Do we have to use this script differently now? Like, will this have to change?
https://github.com/acl-services/sriracha/blob/master/src/GlobalNavigation/package.json#L18

Otherwise, it's not really a `major` (breaking) change, is it?